packages/front-end/components/Features/CodeSnippetModal.tsx

```diff
@@ -337,20 +338,22 @@ export default function CodeSnippetModal({
                 className="cursor-pointer"
                 onClick={(e) => {
                   e.preventDefault();
-                  setInstallationOpen(!installationOpen);
+                  setSetupOpen(!setupOpen);
                 }}
               >
-                Installation{" "}
-                {installationOpen ? <FaAngleDown /> : <FaAngleRight />}
+                Setup {setupOpen ? <FaAngleDown /> : <FaAngleRight />}
               </h4>
-              {installationOpen && (
+              {setupOpen && (
```

Comment:
I don't think we should swap this order.  Instead, just move the event tracker selector to the installation step for the HTML Script Tag SDK.